modules/oe_starter_content_news/config/install/field.field.node.oe_news.oe_featured_media.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_featured_media
+    - node.type.oe_news
+id: node.oe_news.oe_featured_media
+field_name: oe_featured_media
+entity_type: node
+bundle: oe_news
+label: 'Featured Media'
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  handler: 'default:media'
+  handler_settings: {  }
```

Comment:
Let's add here the oe_media and oe_media_avportal image bundles. In order to do so, we need to put these modules as dependencies.